index.html

```diff
@@ -140,25 +144,25 @@ <h2><em> Connect: </em></h2>
             <section id ="leave_a_message">
                 <h2><em> Leave a Message </em></h2>
                 <div class ="container-m">
-                    <!--<form action="action_page.php">-->
-                     <form action="leave_message">
+                
+                    <form action="leave_message">
```

Comment:
I believe this should have the name attribute as `leave_message`. Not a big deal.